src/components/grid/__test__/NodeGrid-test.js

```diff
@@ -1,34 +1,38 @@
-import { assert, React, ReactTestUtils } from '../../../../test/setup';
+/* import { Provider, assert, React, ReactTestUtils } from '../../../../test/setup';
```

Comment:
nope. add it back and figure out how to make them work. We can look at this tomorrow